akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -45,45 +46,53 @@ private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with Stage
 
       class OutStream(
         val streamId:           Int,
-        private var maybeInlet: Option[SubSinkInlet[ByteString]],
+        private var maybeInlet: Option[SubSinkInlet[_]],
         var outboundWindowLeft: Int,
-        private var buffer:     ByteString                       = ByteString.empty,
-        var upstreamClosed:     Boolean                          = false,
-        var endStreamSent:      Boolean                          = false
+        private var buffer:     ByteString                 = ByteString.empty,
+        var upstreamClosed:     Boolean                    = false,
+        var endStreamSent:      Boolean                    = false,
+        var trailer:            Option[ParsedHeadersFrame] = None
```

Comment:
would at some point change all the Option's here to OptionVals, but keeping as is for now is fine